doc/source/advanced.md

```diff
@@ -75,6 +75,70 @@ and **google cloud's ingress controller**.
 This should provision a certificate, and keep renewing it whenever it gets close
 to expiry!
 
+
+## Using PostgreSQL as database for you Hub.
+
+Four types of settings are allowed for storing hub state, as described in
+the key ``hub.db.type``:
+
+- ``sqlite-pvc``
+- ``sqlite-memory``
+- ``mysql``
+- ``postgres``
+
+Using PostgreSql or MySQL provides the main advantage to allow seemless update
+of your hub without discontinuity of service (logged in users can always
+continue to work on running server, even with SQLite, but the difference between
+SQLite and MySQL/PostgreSQL are for new user, in order to allow them to log in
+even when a new hub is starting).
+
+Please note running PostgreSQL on Kubernetes is hard. You can find an lot of
+detailed articles on the web about this suject, such as: 
+[PostgreSQL on Kubernetes the Right Way](https://medium.com/kokster/postgresql-on-kubernetes-the-right-way-part-one-d174ee8a56e3)
+
+It is advised to use the Database as a Service solution that your Cloud
+provider may provide you.
+
+You have the option to run PostgreSQL for the Hub using Helm 'stable/postgresql'
+chart. Use it with full knowledge of its limitations, and that it may probably
+not be suited for an High Available JupyterHub.
+
+Here is an example of configuring PostgreSQL as Hub database using Helm:
+
+```bash
+$ cat > pg-values.yaml <<EOF
+postgresUser: jupyterhub
+postgresPassword: jupyterhub
+postgresDatabase: jupyterhub
+persistence:
+  enabled: False  # see helm 'stable/postgresql` documentation if you want pesistance
+EOF
+$ helm upgrade --install --name hub-db -f pg-values.yaml stable/postgresql
+```
+
+And then set `hub.db.type` to:
+
+```yaml
+hub:
+  db:
+    type: postgresql
+    url: postpostgres+psycopg2://jupyerhub:jupyterhub@hub-db-postgresql:5432/jupyterhub
+```
+
+Note the following points when using PostgreSQL as database for the Hub:
+
+- If you do not use a persistant volume on PostgreSQL, your database will be
```

Comment:
This only applies if the postgres database is set up with helm, which we should recommend against. Possibly remove this whole section?

Reply:
i would not choose for the user, I prefer giving him the information, the limitation of each solution, so he can make his own choices depending on his own situation.

We are dealing with people that are deploying JH in a kubernetes cluster, this population should know what they are doing. It is always better to give them hints and links to more informaiton, and I learn a lot when reading zero2jupyterhub documentation.

I guess you would like to prevent people from making mistakes, that's honorable, but if i would write a scrum story for this feature I would formulate it such as: "as a devops that would like to deploy jupyterhub in my kubernetes cluster, I would like to be able to use a postgresql server started on my cluster, when I do not have an DBaaS option"